<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChartForm</class>
 <widget class="QWidget" name="ChartForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>912</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Results</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="0">
    <widget class="QTreeView" name="tAnswers"/>
   </item>
   <item row="3" column="0">
    <widget class="QTableView" name="tBalance"/>
   </item>
   <item row="1" column="1" rowspan="3">
    <layout class="QGridLayout" name="gridLayout"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lTotal">
     <property name="text">
      <string>Total: </string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="lQuestion">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
